Client/WebFilters/DebouncingResourceDownloader.swift

```diff
@@ -239,6 +239,7 @@ class DebouncingResourceDownloader {
     } else {
       return URL(string: "https://adblock-data-staging.s3.bravesoftware.com")!
     }
+    return URL(string: "#")!
```

Comment:
The lines right above this (for `baseResourceURL`; 236-240) still have `brave.com` for the endpoint. Can you please update this?